### PR TITLE
IR-2872 compress image menu layout fixes

### PR DESCRIPTION
**Pull request by SamMazerIR**
menu items were being squished horizontally and taking up two vertical lines, looked bad and didn't match figma, fixing

## Summary
image compression design changes (fix)
menu items were being squished horizontally and taking up two vertical lines, looked bad and didn't match figma, fixing

problem screenshot (from mt-dev/dev):
current dev/mt-dev

fixed screenshot:
after my adjustments

and if you're crazy enough to use this in a thinner screen:
400x924

## References
[https://tsu.atlassian.net/browse/IR-2872](https://tsu.atlassian.net/browse/IR-2872)

[figma reference](https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=6149-9649&t=D3reKkRs0GeFUqVl-0)

figma screenshot:
figma screencap

## QA Steps
go to files tab and select an image, right click it and select "compress" to bring up menu

### CODE REVIEW

File: packages/editor/src/components/assets/ImageCompressionPanel.tsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ export default function ImageCompressionPanel({
         />
       </div>
 
-      <div className="mx-auto grid w-1/2 gap-y-2">
+      <div className="mx-auto grid w-[80%] justify-center gap-y-2">
```
Comment (aditya-mitra) [inline packages/editor/src/components/assets/ImageCompressionPanel.tsx]:
```suggestion
      <div className="mx-auto grid w-4/5 justify-center gap-y-2">
```

**Review by aditya-mitra — DISMISSED** [review]
can you also check if it retains this width on smaller and larger screens?

**Comment by SamMazerIR** [comment]
> can you also check if it retains this width on smaller and larger screens?

the popup is set to a fixed 680px wide (not my change)

**Review by aditya-mitra — APPROVED** [review]
awesome 🚀

**Pull request merged**